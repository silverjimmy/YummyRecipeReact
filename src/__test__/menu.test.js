import React from 'react';
import ReactDOM from 'react-dom';
import { render,  configure,shallow } from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import expect from "expect";
import Adapter from 'enzyme-adapter-react-15';
import Menu from '../components/menu';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';


 configure({ adapter: new Adapter() });


describe('<Menu />', () => {
    it('matches  snapshot', () => {
            const header = shallow(<Menu />);
           expect(shallowToJson(header)).toMatchSnapshot();
        });

})