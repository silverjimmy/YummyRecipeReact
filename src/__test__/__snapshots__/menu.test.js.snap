// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> matches  snapshot 1`] = `
<div
  className="inner"
>
  <AppBar
    iconElementRight={
      <div>
        <Link
          activeClassName="active"
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/register"
        >
          <RaisedButton
            disabled={false}
            fullWidth={false}
            label="Register"
            labelPosition="after"
            primary={true}
            secondary={false}
            style={
              Object {
                "cursor": "pointer",
                "margin": 12,
              }
            }
          />
        </Link>
        <RaisedButton
          disabled={false}
          fullWidth={false}
          label="LogOut"
          labelPosition="after"
          onClick={[Function]}
          primary={false}
          secondary={true}
          style={
            Object {
              "cursor": "pointer",
              "margin": 12,
            }
          }
        />
      </div>
    }
    showMenuIconButton={true}
    title={
      <span
        style={undefined}
      >
        <Link
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/dashboard"
        >
          YummyRecipes
        </Link>
      </span>
    }
    zDepth={1}
  />
</div>
`;
